### DISCUSSION TITLE
Recommended image editor

### DISCUSSION THREAD

**Discussion by Fastering18** [flame-engine/flame#936] [Q&A] [answered]
I wonder if there any image editor that designed for manipulating textures like  
[2nd tutorial robot.png](https://github.com/flame-engine/flame/blob/main/tutorials/2_sprite_animations_gestures/code/assets/images/robot.png?raw=true) ![2nd tutorial robot.png](https://raw.githubusercontent.com/flame-engine/flame/main/tutorials/2_sprite_animations_gestures/code/assets/images/robot.png)   

Any website/app image editor recommendations?

Edit: [Aseprite](https://www.aseprite.org/trial) won't let me to export file for trial version, is there any alternative(s)?

**Comment by erickzanardo** [comment, marked as answer]
I really recommend aseprite, that robot was manipulated on it

**Reply by Fastering18** [reply]
Thanks🙏 ill try it out!

**Reply by Fastering18** [reply]
[Aseprite](https://www.aseprite.org/trial) doesn't let me to export file for trial version, is there any alternative(s)?

**Reply by erickzanardo** [reply]
None that I am familiar with

**Comment by nickmeinhold** [comment]
Perhaps this youtube video is helpful?  

[What Program to use for Pixel Art? (Paid and Free Software)](https://www.youtube.com/watch?v=90BghUX7SD0&t=331s)
